src/win/fs.c

```diff
@@ -126,7 +126,7 @@ static int uv__usermode_symlink_available(void) {
   }
   pRtlGetVersion(&version_information);
   /* Usermode symlinks are only available on Win10 with Creators Update... */
-  if (version_information.dwMajorVersion < 10 ||
+  if (version_information.dwMajorVersion != 10 ||
```

Comment:
We don't really know how win11 will behave. IMHO it is better to play it safe and enable this on win11 when it comes out and we know how it works, than to break node and other apps on that platform.